src/components/Game.js

```diff
@@ -196,10 +196,10 @@ function Game() {
     <div className="main-container">
       <div className="top-header">
         <img
+        className="storylingo-img"
```

Comment:
the class name should be specific

Reply:
I changed the name. Now it's storyling-logo. 
or You can suggest me.